values.yaml

```diff
@@ -279,26 +306,69 @@ nifi:
   persistence:
     enabled: true
   properties:
-    externalSecure: false
-    isNode: true
-    httpPort: 8080
-    httpsPort: null
-    webProxyHost: # set it by the same host than nifi.traefikIngress.host if traefik ingress is enabled
+    sensitiveKey: changeMechangeMe # Must have at least 12 characters
+    isNode: false
+    httpsPort: 8443
+    webProxyHost: <your_nifi_domain_name> # set it by the same host than nifi.traefikIngress.host if traefik ingress is enabled
     clusterPort: 6007
-    clusterSecure: false
+    customLibPath: "/opt/configuration_resources/custom_lib"
   auth:
+    admin: CN=admin, OU=NIFI
+    SSL:
+      keystorePasswd: changeMe
+      truststorePasswd: changeMe
+  
+  # Automaticaly disabled if OIDC or LDAP enabled 
+    singleUser:
+      username: username
+      password: changemechangeme # Must to have at least 12 characters
+
     ldap:
-      enabled: true
+      enabled: false
```

Comment:
ldap should be enabled by default.
If that is not possible, document why in an issue.